### PR TITLE
HM-06 Added new methods to processor.ts and 3 new test in consent.test.ts

### PR DESCRIPTION
**Pull request by mon-bene**

### CODE REVIEW

File: src/processor.ts [hunk 1]
```diff
@@ -8,4 +8,16 @@ export class Processor {
...
+
+  checkAge(userInput: User, age: number): void {
+    if (age >= 18) {
+      userInput.consentGiven = true
+    } else {
+      userInput.consentGiven = false
+    }
+  }
+
+  revokeConsent(userInput: User): void {
```
Comment (vladimir-tl) [inline src/processor.ts:20]: If you intend to revoke consent, it might make more sense to set consentGiven to false

File: test/consent.test.ts [hunk 2]
```diff
@@ -22,4 +22,17 @@ describe('Test for consent', () => {
...
+
+  test('A user younger than 18 cannot give consent', () => {
+    processor.checkAge(user, 12)
+    expect(user.consentGiven).toBe(false)
+  })
+  test('A user older than 18 can successfully give consent', () => {
+    processor.checkAge(user, 18)
+    expect(user.consentGiven).toBe(true)
+  })
+  test('verify that it is correctly revokes the consent', () => {
```
Comment (vladimir-tl) [inline test/consent.test.ts:34]: then test should be updated

**Review by vladimir-tl — CHANGES_REQUESTED** [review]

File: src/processor.ts (hunk 1 above)
Comment (mon-bene) [reply src/processor.ts:20]: resolved -> changed to false

File: test/consent.test.ts (hunk 2 above)
Comment (mon-bene) [reply test/consent.test.ts:34]: test updated (chnaged also to false)

**Pull request merged**